### ISSUE TITLE
Pending boss damage displays in scientific notation

### ISSUE THREAD

**Issue by JenniferMartin** [HabitRPG/habitica#7980]
I'm not sure if this was deliberate or not, but the new pending boss damage "target" displays the amount of pending damage in scientific notation once past a certain threshold. (I'm not sure what the exact threshold is--see below.) I personally find standard notation to be significantly easier to read, and I wouldn't be surprised if there are a significant number of people who would not recognize the number as scientific notation and/or who can't read scientific notation. Also, standard notation would be shorter for the amounts of damage Habitica users are likely to be seeing.

How I noticed this: When I first noticed the new "target", I had done 20 damage towards the boss, reported as "20" pending damage (or possibly "+20"). After checking off some tasks and using skills, I checked again, and it reported that I had "+2.2e+2" pending damage accumulated--that is, 220 damage.

In summary: I am suggesting that the damage displayed in the pending boss damage counter always be displayed in standard notation, not scientific notation.

**Comment by crookedneighbor** [comment]
Agreed. I don't think i was intentional to display as scientific notation.

**Comment by crookedneighbor** [comment]
Will go out in the next deploy.
